website/gamesOf.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+use Carbon\Carbon;
+use Cocur\HumanDate\HumanDate;
+
+$config = require '/var/creds/creds.php';
+$db = $config['database'];
+
+$conn = mysqli_connect($db['host'], $db['user'], $db['password'], $db['dbname']);
+if (!$conn) {
+    die('Connection failed: ' . mysqli_connect_error());
+}
+
+$n = $_GET['name'];
+$offset = (int)$_GET['offset'];
+
+$sql = "select * from games where player1_name='$n' or player2_name='$n' or player3_name='$n' or player4_name='$n' or player5_name='$n' or player6_name='$n' and status='completed' order by time_start desc limit 10 offset $offset";
```

Comment:
If this goes in at the same time as the schema changes the query will need to be updated. Should be able to just use a join to the game_players table

`SELECT * from games g join game_players gp on g.id == gp.game_id` 

I can help update the queries before we go live with the schema changes too